#!/usr/bin/python3


def raise_exception_msg(message=""):
    """raises the name exception with a message."""
    raise NameError(message)
